datahub_sap_hana/ingestion.py

```diff
@@ -178,3 +203,178 @@ def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
                 wu = item.as_workunit()
                 self.report.report_workunit(wu)
                 yield wu
+
+    def get_column_lineage_view_definitions(self, conn) -> Iterable[Tuple[str, str]]:
+        with conn:
+            inspector = inspect(conn)
+            schema: List[Any] = inspector.get_schema_names()  # returns a list
+
+            for schema_name in schema:
+                views = inspector.get_view_names(schema=schema_name)  # returns a list
+
+                for view_name in views:
+                    view_sql = inspector.get_view_definition(view_name, schema_name)
+
+                    if view_sql:
+                        yield (view_name, view_sql)
+
+    def _get_column_lineage_for_view(
+        self, view_name: str, view_sql: str
+    ) -> Tuple[str, List[Node]]:
+        """Extracts the columns and the sql definitions of a downstream view.
+        These are then passed to the lineage function create a lineage graph of the column of an sql query.
```

Comment:
it is worth pointing out that the lineage function comes from a package
```suggestion
        These are then passed to the sqlglot lineage function create a lineage graph of the column of an sql query.
```